source/contentprovider.d

```diff
@@ -82,6 +90,12 @@ class ContentProvider
 			if (parts[1] != "index.yml")
 				continue;
 			auto root = Loader(filename).load();
+
+			// language meta information
+			enforce("start" in root, "'start' point required in language-specific yaml");
+			enforce("title" in root, "'title' point required in language-specific yaml");
```

Comment:
Please enforce that start is made out of two parts separated by /. Otherwise we might end up with a range exception at runtime when `index()` is called in `WebInterface`